block/header.go

```diff
@@ -148,26 +148,30 @@ func (h *Header) SigningHash() (hash thor.Bytes32) {
 	defer func() { h.cache.signingHash.Store(hash) }()
 
 	return thor.Blake2bFn(func(w io.Writer) {
-		hashBody := []any{
-			&h.body.ParentID,
-			h.body.Timestamp,
-			h.body.GasLimit,
-			&h.body.Beneficiary,
-
-			h.body.GasUsed,
-			h.body.TotalScore,
-
-			&h.body.TxsRootFeatures,
-			&h.body.StateRoot,
-			&h.body.ReceiptsRoot,
-		}
-		if h.body.Extension.BaseFee != nil {
-			hashBody = append(hashBody, &h.body.Extension.BaseFee)
-		}
-		rlp.Encode(w, hashBody)
+		rlp.Encode(w, h.signingFields())
 	})
 }
 
+func (h *Header) signingFields() []any {
+	fields := []any{
+		h.body.ParentID,
```

Comment:
Question: Why some of the fields do not have the reference (`&`) anymore?

Reply:
`thor.Bytes32 [32]byte` both works with and without pointer access when doing the RLP encoding, in other places, we are not using pointers for `thor.Bytes32` so just removed them here.

Reply:
Perfect, thanks for clarifying!